CommandLine/Accept.cs

```diff
@@ -86,6 +86,29 @@ public static class Accept
                 return null;
             }));
 
+        public static ArgumentsRule LegalFilePathsOnly(
```

Comment:
The order of the arguments rules when chaining them is arbitrary, right? In that case, shouldn't all of them have "with" so they read the same?